src/com_tjvendors/admin/models/forms/vendorfee.xml

```diff
@@ -29,7 +29,7 @@
 
 		<field
 			name="currency"
-			type="text"
+			type="currencylist"
```

Comment:
@Sachintechjoomla DO we need to consider any kind of migration for this?
If we have already created fees for currency USD then will USD be pre-selected after the code change? Also if a user has added a currency that is not there in the list then what will happen?